### DISCUSSION TITLE
Applying tax

### DISCUSSION THREAD

**Discussion by khizar-wahab** [fattureincloud/api#288] [Q&A] [answered]
I am creating and invoice to which items and payments will be added later. I need to apply 10% tax to the total, i have tried withholding_tax fields but they deduct tax from total rather than adding. What fields should i use for applying tax.
I also tried "amount_vat" but does not work.

**Comment by emajo** [comment, marked as answer]
Hi @khizar-wahab, it's not possible to do what you are trying to do, you must set at least an item with the related `amount_net` to be able to set the `vat` field, I suggest to check out the [Invoice creation guide](https://developers.fattureincloud.it/docs/guides/invoice-creation) to understand better how it works.

Thanks,
Emanuele.

**Reply by khizar-wahab** [reply]
The vat field in item applies tax individually to each item. I need to apply tax to the total once. For example i have 3 items in the invoice. Each item has price of 1200 which makes the total 3600. Now i want to apply 22% tax on 3600.
